specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/IntegrationRuntime.json

```diff
@@ -134,6 +134,18 @@
         "catalogInfo": {
           "description": "Catalog information for managed dedicated integration runtime.",
           "$ref": "#/definitions/IntegrationRuntimeSsisCatalogInfo"
+        },
+        "licenseType": {

```

Comment:
@HaoQian-MS Is this possible to update one of the IntegrationRuntime example to show this new field?

@olydis @fearthecowboy should all enum be "extensible" now? Is this the new global recommendation or depends on the scenario?

Reply:
@lmazuel -- leave it at `modelAsString` for the moment; c# is the only one that does anything with extensible, and I would need to put a directive in place to fix it for other generators.

Reply:
@lmazuel Example added.